packages/input-selection/test/InputSelectionPropertyTesting.test.ts

```diff
@@ -71,28 +74,48 @@ const testInputSelection = (name: string, getAlgorithm: () => InputSelector) =>
           mockConstraints: SelectionConstraints.MOCK_NO_CONSTRAINTS
         });
       });
+
+      it('includes must spend inputs', async () => {
```

Comment:
This seems to be missing an assertion that selection includes the pre-selected utxo (mustSpendUtxo)

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1299/commits/2967bf1faee5b56d191749acd55c9a37c2add29e